src/ocrd_utils/ocrd_logging.conf

```diff
@@ -56,30 +56,30 @@ handlers=consoleHandler,fileHandler
 # ocrd loggers
 [logger_ocrd]
 level=INFO
-handlers=consoleHandler,fileHandler
+handlers=
 qualname=ocrd
-propagate=0
 
 [logger_ocrd_network]
 level=INFO
-handlers=consoleHandler,processingServerHandler
+#handlers=consoleHandler,processingServerHandler
+handlers=processingServerHandler
 qualname=ocrd_network
-propagate=0
+#propagate=0
 
 #
 # logger tensorflow
 #
 [logger_ocrd_tensorflow]
 level=ERROR
-handlers=consoleHandler
+handlers=
```

Comment:
What are the semantics of setting handlers to an empty value? It's not ovious from https://docs.python.org/3/library/logging.config.html for me - I assume it's incremental, i.e. does not explicitly add a handler but does not remove existing handlers either?

Reply:
Depends on how `fileConfig` gets run. The default is `disable_existing_loggers=True`, i.e. before parsing, all loggers (and their handlers) are removed, as done in our `disableLogging`. See [documentation](https://docs.python.org/3/library/logging.config.html#logging.config.fileConfig).

Still, what was surprising to me is that one _must_ add a `handlers` statement, it cannot be omitted – even if one just wants the messages to get forwarded up in the hierarchy without extra handlers.

The config file is documented [here](https://docs.python.org/3/library/logging.config.html#logging-config-fileformat) BTW.